.coveragerc

```diff
@@ -12,6 +12,9 @@ source =
 [report]
 # Regexes for lines to exclude from consideration
 exclude_lines =
+    # Don't complain about ellipsis (used in overloaded methods)
+    \.\.\.
```

Comment:
So, this line in particular is motivate by the `@overload` decorator in the standard lib.
In all [examples there](https://docs.python.org/3/library/typing.html#typing.overload) they use ellipsis instead of `pass`, so a person starting to use this decorator would naturally use the ellipsis.

I suppose we could replace with `pass`, but I confess that I never thought that I could do that. I just learned the entire idiom from the docs without thinking about its parts 😝

Reply:
Okay, then let's stay with `...` if that's the default.